src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -39,14 +37,18 @@ public class BlobFileStore : IFileStore
         IOptions<BlobOperationOptions> options,
         ILogger<BlobFileStore> logger)
     {
-        _nameWithPrefix = EnsureArg.IsNotNull(nameWithPrefix, nameof(nameWithPrefix));
+        NameWithPrefix = EnsureArg.IsNotNull(nameWithPrefix, nameof(nameWithPrefix));
         _options = EnsureArg.IsNotNull(options?.Value, nameof(options));
         _logger = EnsureArg.IsNotNull(logger, nameof(logger));
-        _blobClient = EnsureArg.IsNotNull(blobClient, nameof(blobClient));
+        BlobClient = EnsureArg.IsNotNull(blobClient, nameof(blobClient));
     }
 
+    protected DicomFileNameWithPrefix NameWithPrefix { get; }
+
+    private IBlobClient BlobClient { get; }
```

Comment:
nit: use fields for private members unless there is some more complex logic on get/set